main/exercise/UniqueAnswerImage.php

```diff
@@ -59,6 +59,64 @@ public function createAnswersForm($form)
 
         $html = '<div class="alert alert-success" role="alert">'.
                 get_lang('UniqueAnswerImagePreferredSize200x150').'</div>';
+        
+        $zoomOptions = api_get_configuration_value('zoom_images_quiz');
+        if (isset($zoomOptions['options'])) {
+            $finderFolder = api_get_path(WEB_PATH).'vendor/studio-42/elfinder/';
+            $html .= '<!-- elFinder CSS (REQUIRED) -->';
+            $html .= '<link rel="stylesheet" type="text/css" media="screen" href="'.$finderFolder.'css/elfinder.full.css">';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line exceeds maximum limit of 120 characters; contains 124 characters](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5015949)